src/main.ts

```diff
@@ -377,6 +384,24 @@ getResultsButton.forEach((getResult) => {
   });
 });
 
+// event listener for animation inputs to display animation preview
+const attribute = 'animation';
```

Comment:
i think it's also unnecessary because this could cause a bug, in the future. You can just use the text directly